PerformanceCalculator/Difficulty/DifficultyCommand.cs

```diff
@@ -23,9 +24,9 @@ namespace PerformanceCalculator.Difficulty
     public class DifficultyCommand : ProcessorCommand
     {
         [UsedImplicitly]
-        [Required, FileOrDirectoryExists]
-        [Argument(0, Name = "path", Description = "Required. A beatmap file (.osu), or a folder containing .osu files to compute the difficulty for.")]
-        public string Path { get; }
+        [Required]
+        [Argument(0, Name = "path", Description = "Required. A beatmap file (.osu), beatmap ID, or a folder containing .osu files to compute the difficulty for.")]
+        public string Path { get; private set; }
```

Comment:
```suggestion
[UsedImplicitly]
        [Required]
        [Argument(0, Name = "beatmap", Description = "Required. Can be either a path to beatmap file (.osu) or beatmap ID.")]
        public string Beatmap { get; set; }
```

Use the same as used in other implementations. It can't be called path if it isn't always a path.

Reply:
You completely changed the semantics of this. I want to keep it as a path.

Reply:
This command allows a whole directory to be processed, while others only take singular beatmaps

Reply:
Super messy. I'm just going to turn a blind eye.